generator/src/main/java/org/corfudb/generator/operations/CheckpointOperation.java

```diff
@@ -28,10 +29,10 @@ public void execute() {
 
             MultiCheckpointWriter mcw = new MultiCheckpointWriter();
             mcw.addAllMaps(state.getMaps());
-            long trimAddress = mcw.appendCheckpoints(state.getRuntime(), "Maithem");
-            state.setTrimMark(trimAddress);
+            Token trimAddress = mcw.appendCheckpoints(state.getRuntime(), "Maithem");
+            state.setTrimMark(trimAddress.getSequence());
```

Comment:
Is it somehow cleaner to reference the trim mark as a token rather than just a sequence number?